ops/main.py

```diff
@@ -156,6 +156,73 @@ def _get_event_args(charm, bound_event):
     return [], {}
 
 
+class _JujuEvent:
```

Comment:
better name wanted

Reply:
OTOH we could move all the other helpers under this and then the name would make sense :-)